### PR TITLE
Update MLP.md

### PR DESCRIPTION
**Pull request by NxdsLjk**
主要对最后部分的代码进行了修改：
28：torch.flatten 的参数问题： torch.flatten 的正确用法是torch.flatten(input, start_dim=0, end_dim=-1)，仅用‘-1’作为参数会报错，这里修改为x = torch.flatten(x, start_dim=1)。 30：nn.Softmax 是一个类，而不是一个函数，这里修改为nn.functional.softmax()。 68：在较新的 PyTorch 版本中，loss.data[0] 这种访问方式已经被弃用，使用 loss.item() 来获取损失值。 92：torch.argmax(pred) 返回的是整个张量中最大值的索引，而不是每个样本的最大值索引，应该使用 torch.argmax(pred, dim=1) 来获取每个样本的预测类别。

### CODE REVIEW

**Comment by ProbiusOfficial** [comment]
@CainYzb
